src/app/home-page/widget/filtered-list.ts

```diff
@@ -14,17 +15,25 @@ export abstract class FilteredList extends Base implements OnInit {
   public myItems;
   public filterText;
   public isLoading = true;
+  private subject: Subject<string> = new Subject();
 
   constructor(protected userQuery: UserQuery, protected entriesService: EntriesService, protected usersService: UsersService) {
     super();
   }
 
   ngOnInit() {
     this.getMyList();
+
+    this.subject.pipe(
+      // formDebounceTime is set to 250 which is not long enough
+      debounceTime(500)
```

Comment:
also need to unsubscribe and https://www.learnrxjs.io/learn-rxjs/operators/filtering/distinctuntilchanged